hdfs3/tests/test_hdfs3.py

```diff
@@ -159,6 +159,15 @@ def test_write_blocksize(hdfs):
         hdfs.open(a, 'rb', block_size=123)
 
 
+def test_write_vbig(hdfs):
+    with hdfs.open(a, 'wb') as f:
+        f.write(b' ' * 2**31)
```

Comment:
Longer than I thought.... ~60s for the one test (including launching py.test), ~90s for everything. I'll mark it as `@slow`, but how do I choose to run it in the future?